### PR TITLE
Switch to io.smallrye.certs:smallrye-certificate-generator-junit5

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

**Review by aldettinger — APPROVED** [review]

File: poms/bom-test/pom.xml [hunk 1]
```diff
@@ -81,9 +81,9 @@
                 <version>${quarkiverse-micrometer.version}</version>
             </dependency>
             <dependency>
-                <groupId>me.escoffier.certs</groupId>
-                <artifactId>certificate-generator-junit5</artifactId>
-                <version>${certificate.generator.version}</version>
+                <groupId>io.smallrye.certs</groupId>
+                <artifactId>smallrye-certificate-generator-junit5</artifactId>
+                <version>${smallrye-certificate-generator-junit5.version}</version>
```
Comment (ppalaga) [inline poms/bom-test/pom.xml:86]: Could you please add a comment to remove this once https://github.com/quarkusio/quarkus/pull/43393 reaches us?

File: pom.xml [hunk 2]
```diff
@@ -172,7 +172,7 @@
         <zstd-jni.version>1.5.6-3</zstd-jni.version><!-- @sync org.apache.kafka:kafka-clients:${kafka.version} dep:com.github.luben:zstd-jni -->
 
         <!-- Test dependency versions (keep sorted alphabetically) -->
-        <certificate.generator.version>0.7.1</certificate.generator.version>
+        <smallrye-certificate-generator-junit5.version>0.8.1</smallrye-certificate-generator-junit5.version>
```
Comment (ppalaga) [inline pom.xml:175]:
```suggestion
        <smallrye-certificate-generator-junit5.version>0.8.1</smallrye-certificate-generator-junit5.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:io.smallrye.certs:smallrye-certificate-generator -->
```
To be sure to stay in sync with Quarkus BOM. See also https://github.com/quarkusio/quarkus/pull/43393

**Review by ppalaga — APPROVED** [review]

File: poms/bom-test/pom.xml (hunk 1 above)
Comment (jamesnetherton) [reply poms/bom-test/pom.xml:86]: I didn't bother because I'm planning to remove it immediately on `quarkus-main`.

File: pom.xml (hunk 2 above)
Comment (jamesnetherton) [reply pom.xml:175]: Ah, thanks. I missed it was already part of Quarkus 3.15.0.

I'll add the sync bits.

File: poms/bom-test/pom.xml (hunk 1 above)
Comment (ppalaga) [reply poms/bom-test/pom.xml:86]: Good idea :)

File: pom.xml (hunk 2 above)
Comment (jamesnetherton) [reply pom.xml]: Latest commit adds `@sync`.

File: poms/bom-test/pom.xml (hunk 1 above)
Comment (jamesnetherton) [reply poms/bom-test/pom.xml:87]: I added a comment as suggested in the latest commit.

**Pull request merged**